TenantFile/Api/Controllers/SmsController.cs

```diff
@@ -41,29 +42,51 @@ public SmsController(ILogger<SmsController> logger, ICloudStorage storageClient,
             this.keyWords = new List<string>() { };
             this.keyPhrases = new List<string>() { "written request for repairs" };
         }
-
-        [HttpPost("/api/captureNew")]
+        [HttpPost("/api/captureData")]
         public async Task<IActionResult> CaptureTenantData([FromBody] FlowData data)
         {
 
-            TryGetPhoneEntity(out var phone, data.from);
+            //this should never be a newPhone even if this encounter is the first time the Tenant has sent an image because the new Phone would already have been captured and persisted in the early part of the Flow
+            bool newPhone = GetOrCreatePhone(out var phone, data.from);
 
-            var tenant = context.Tenants.AsQueryable().Where(t => t.Phones.Contains(phone)).First();
+            var property = context.Properties.Include(p => p.Address).AsQueryable().Where(p => p.Name == data.propertyName).First();
 
-            tenant.Name = string.Join(" ", data.firstName, data.lastName);
-            tenant.CurrentResidence = new Residence()
+            if (!newPhone)
```

Comment:
Wouldn't this always be true? Doesn't newPhone create a new phone if it can't find one? Perhaps you could just the "get" part of getting the phone

Reply:
@ryanrightmer Yeah that should be the case, made the change in the next commit